apps/etterlatte-behandling/src/main/resources/db/migration/V62__etterbetaling.sql

```diff
@@ -0,0 +1,9 @@
+create table etterbetaling
+(
+    behandlings_id UUID PRIMARY KEY
+        CONSTRAINT etterbetaling_behandling_fk_id
+            REFERENCES behandling (id),
+    opprettet   TIMESTAMP,
+    fra_dato DATE,
```

Comment:
Burde vi bare bruke `DATE` og ikke `TIMESTAMP`?

Reply:
> Burde vi bare bruke `DATE` og ikke `TIMESTAMP`?

Tenkjer dei har forskjellige formål. For fradato er ikkje tidspunktet relevant, medan når vi oppretta innslaget i databasen kan vera nyttig for oss teknikarar å veta.